spec/player_spec.rb

```diff
@@ -19,11 +19,28 @@ def input
       @inputs.shift
     end
   end
+
+  class Player::FakeBoard
+    attr_reader :board
+
+    def initialize
+      @board = Array.new(9)
+    end
+
+    def place_token(position)
+      board[position - 1] = "X"
+    end
+
+    def is_available?(position)
+      board[position - 1].nil?
+    end
+  end
```

Comment:
This fake also looks suspicious. It is very similar to the real board.

Reply:
I'll simplify